### PR TITLE
Move docs creation to CI

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR removes the `docs` folder and make some adjustments in order to configure the Jenkins pipeline to build and push the compiled docs. This is better for mantainabilty as developers will only have to update the source content.

### CODE REVIEW

File: deploy/local/live-csc/.env [hunk 1]
```diff
@@ -7,35 +7,42 @@ DOCKERFILE_PATH_COMMANDER=../../../../LOVE-commander/
 
 # INTERACTION BETWEEN SERVICES
 SERVER_URL=*
-LOVE_PRODUCER_WEBSOCKET_HOST=love-nginx-mount/manager/ws/subscription
+#LOVE_PRODUCER_WEBSOCKET_HOST=love-nginx-mount/manager/ws/subscription
```
Comment (wvreeven) [inline deploy/local/live-csc/.env:10]: In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.

File: deploy/local/live-csc/docker-compose.yml [hunk 2]
```diff
@@ -16,9 +16,9 @@ x-manager-env: &manager-environment
   SERVER_URL: ${SERVER_URL}
   REDIS_HOST: ${REDIS_HOST}
   REDIS_PASS: ${REDIS_PASS}
-  AUTH_LDAP_1_SERVER_URI: ${AUTH_LDAP_1_SERVER_URI}
-  AUTH_LDAP_2_SERVER_URI: ${AUTH_LDAP_2_SERVER_URI}
-  AUTH_LDAP_3_SERVER_URI: ${AUTH_LDAP_3_SERVER_URI}
+  # AUTH_LDAP_1_SERVER_URI: ${AUTH_LDAP_1_SERVER_URI}
```
Comment (wvreeven) [inline deploy/local/live-csc/docker-compose.yml:19]: Same here. In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.

File: deploy/local/live-csc/nginx.conf [hunk 3]
```diff
@@ -1,14 +1,16 @@
 upstream manager_load_balancer {
-    server love-manager-mount-01:8000;
-    server love-manager-mount-02:8000;
+    server localhost:8000;
+    #server love-manager-mount-01:8000;
```
Comment (wvreeven) [inline deploy/local/live-csc/nginx.conf:3]: And again. In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.

**Review by wvreeven — APPROVED** [review]
Looks good to me. See my comments though.

File: deploy/local/live-csc/nginx.conf (hunk 3 above)
Comment (sebastian-aranda) [reply deploy/local/live-csc/nginx.conf:3]: Oh sorry about this, that was not intended to be pushed. I will remove this part.

**Pull request merged**